db/migrate/20220427004727_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly add the necessary foreign key to the `Likes` table which will create `author_id`, `post_id`, and a corresponding index. The foreign key is on the ERD diagram.